common/src/main/java/tds/common/configuration/EventLoggerConfiguration.java

```diff
@@ -31,6 +31,6 @@ public EventLoggerConfiguration(final ApplicationContext applicationContext,
   @Bean
   public MappedInterceptor eventInterceptor()
   {
-    return new MappedInterceptor(null, new EventLoggerInterceptor(applicationContext.getId(), objectMapper));
+    return new MappedInterceptor(new String[] {"/**"}, new String[] {"/health*"}, new EventLoggerInterceptor(applicationContext.getId(), objectMapper));
```

Comment:
JavaDoc [here](http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/web/servlet/handler/MappedInterceptor.html#MappedInterceptor-java.lang.String:A-java.lang.String:A-org.springframework.web.servlet.HandlerInterceptor-). @jtreuting is correct.

Not that it matters since I believe this is instantiated once per Spring container at startup but both String arrays could be private static variables.